### DISCUSSION TITLE
Walled backup file path

### DISCUSSION THREAD

**Discussion by cpiga** [Chia-Network/chia-blockchain#8461] [Support]
I'm migrating chia from a window host to a linux xen vm (debian). I have the full node synced but not the wallet (height:0 for different days). I stopped, deleted the file in /mainnet/wallet/db and restarted the daemon. Now with "chia wallet show" I get:
>>No online backup file found,
>>Press S to skip restore from backup
>>Press F to use your own backup file:
I have a backup of the mainnet folder of the windows machine but I don't understand which is the backup file... the "./wallet/db/blockchain*.sqlite" file? the path to wallet directory? the path to db directory? 
Please help me
P.S. I know I can skip the backup but I would like to go through the backup procedure to speed up the process...

**Comment by loppefaaret** [comment]
yes, you need to "activate" a wallet when doing CLI - and first time you run a wallet, it ask if you want to import a backup of metadata that might have been saved previously - just `S` to skip that, the wallet will show it status, and is now "active" and will start syncing from the node

the backup feature is _not_ a database backup function - it is purely for advanced wallet setups, with rate limited wallets, CATs or whatever else of advanced wallet setup you might create - in the future - those features are not yet fully accessible from the current client!

**Comment by cpiga** [comment]
I have a backup of the mainnet folder of the windows machine but I don't understand which is the backup file... the "./wallet/db/blockchain*.sqlite" file? the path to wallet directory? the path to db directory?
Please help me
P.S. I know I can skip the backup but I would like to go through the backup procedure to speed up the process...

**Reply by loppefaaret** [reply]
the database can be inserted into the same path as the client it was taken from: `~/.chia/mainnet/db/` for the blockchain, and `~/.chia/mainnet/wallet/db/` for wallets... 
the import backup for wallets is not something to think about yet - i dont really know how else to explain it, than what i have done above... your only option for now, is to `S` to skip!

**Comment by BrandtH22** [comment]
@cpiga , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
